project/classic/dispatcher/image/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM rockylinux:8
```

Comment:
Please add Readme file and explain your core assumptions, e.g. using RHEL-like system, files layout specific to Adobe Managed Services, all the _hacks_ you had to do to make it run.